frontend/android/app/src/main/res/layout/fragment_add_spending.xml

```diff
@@ -34,12 +34,27 @@
             android:layout_marginLeft="12dp"
             android:layout_marginRight="12dp"/>
 
+    <android.support.v7.widget.AppCompatEditText
+            android:id="@+id/dateEditText"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:text="xxxx.xx.xx."
+            app:layout_constraintTop_toBottomOf="@+id/spendingEditText"
+            app:layout_constraintLeft_toLeftOf="parent"
+            app:layout_constraintRight_toRightOf="parent"
+            android:layout_marginTop="12dp"
+            android:layout_marginLeft="12dp"
+            android:layout_marginRight="12dp"
+            android:clickable="true"
+            android:focusable="false"
+            android:inputType="textNoSuggestions"/>
```

Comment:
try date input type maybe, can we use something to add better userexperince to users to know its a date, add label (TextView) or something?